pythia/tasks/scene_graph_database.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+import copy
+
+from pythia.tasks.image_database import ImageDatabase
+
+
+class SceneGraphDatabase(ImageDatabase):
+    def __init__(self, scene_graph_path):
+        super().__init__(scene_graph_path)
+        self.data_dict = {}
+        for item in self.data:
+            self.data_dict[item["image_id"]] = item
+
+    def __getitem__(self, idx):
+        return copy.deepcopy(self.data_dict[idx])
```

Comment:
Why is a `deepcopy` needed here?

Reply:
Changing this to deepcopy inside the dataset instead of here.